openadapt/config.py

```diff
@@ -84,6 +84,9 @@
     ],
 }
 
+# sequences that when typed, will stop the recording of ActionEvents in record.py
+STOP_SEQUENCES = [["o", "a", ".", "s", "t", "o", "p"]]
```

Comment:
Do you think we should use strings here instead? We can then use .split() to achieve the same result, but it'd be easier + faster to define stop sequences this way.

Reply:
```STOP_SEQUENCES = ["oa.stop".split()]```

Reply:
I've made this change and also fixed the listener code to properly support special characters, but the special characters in the stop strings shouldn't have the <> since I'm using the canonical names. Currently, I am splitting the strings by character, so I'm not sure how to split the ctrl c (or any other special characters).

Reply:
I think we would need to specify sequences more explicitly, e.g.:
```
STOP_STRS = [
    "oa.stop",
    ...
]
...
STOP_SEQUENCES = [
    stop_str.split() for stop_str in STOP_STRS
] + [
    ["<ctrl>+c"]
]
```

Note that `<ctrl>+c` denotes both pressed at the same time, whereas `<ctrl>c` would denote one pressed after the other.

Then we would need some extra logic to figure out when these sequences are triggered.